.travis.yml

```diff
@@ -61,24 +55,29 @@ script:
   - >
     echo "Running unit and functional tests…";
     .Build/bin/phpunit;
+jobs:
```

Comment:
Hehe, welcome to the world of build stages. ;-)

Reply:
yeah, pretty nice :)